public/store/AlertsStore.ts

```diff
@@ -58,7 +60,7 @@ export class AlertsStore {
     } while (
       // If we get 10,000 alerts as part of the previous call then there might be more alerts to fetch,
       // hence we make another call until the number of alerts is less then 10,000
-      alertsCount === maxAlertsReturned
+      alertsCount > maxAlertsReturned
```

Comment:
@riysaxen-amzn to double check this doesn't break existing pagination

Reply:
This new condition will never be true. Since alertsCount can never exceed maxAlertsReturned (which is set to 10,000), the loop will effectively terminate after the first fetch.

Reply:
If there are more than 10,000 alerts available, we would miss fetching any alerts beyond the first batch

Reply:
I'm curious about this change as well. The maximum number of alerts that can be returned by the API is 10,000, correct? In that case, I'm not sure it's possible for `alertsCount` to be greater than `maxAlertsReturned`. Whenever the API response is received, `alertsCount` is reassigned to the number of alerts returned on this line
https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/1206/files#diff-34f8a5950df0e00570a3fd41df3207f8de73e3825c3bf21eaead3edee111db12R53

Reply:
Yep reverted the change back, since we are only grabbing 25 alerts max for the threat alerts card, added a change to do this in one call now.